students/arkadiusz_kasprzyk/lesson_02_flow_control/factorial.py

```diff
@@ -0,0 +1,14 @@
+"""
+description: n!
+"""
+
+print("For given n returns n! (factorial).")
+
+n = int(input("n = "))
+factorial = 1
+
+if n > 0:
+    for k in range(1, n+1):
+        factorial *= k
+
+print(factorial)
```

Comment:
If you are checking the input, then this should also be indented or you should provide an else branch to handle the negative numbers.